sql/core/src/test/resources/sql-tests/results/literals.sql.out

```diff
@@ -785,3 +785,59 @@ select -0, -0.0
 struct<0:int,0.0:decimal(1,1)>
 -- !query output
 0	0.0
+
+
+-- !query
+SELECT "S""par""k", "S\"par\"k", 'S""par""k'
```

Comment:
super nit
```suggestion
SELECT "S""par""k" AS c1, "S\"par\"k" AS c2, 'S""par""k' AS c3
```
to make it easier to match the result with the schema.

Reply:
That's nitpicky and reasonable. I fixed them all.